addons/base/utils.py

```diff
@@ -20,3 +21,28 @@ def serialize_addon_config(config, user):
 def get_addons_by_config_type(config_type, user):
     addons = [addon for addon in settings.ADDONS_AVAILABLE if config_type in addon.configs]
     return [serialize_addon_config(addon_config, user) for addon_config in sorted(addons, key=lambda cfg: cfg.full_name.lower())]
+
+def format_last_known_metadata(auth, node, file, error_type):
+    msg = """
+    </div>"""  # None is default
```

Comment:
Where is the opening div?

Reply:
[Here](https://github.com/mfraezz/osf.io/blob/22879db9739afa7abc81a8086d60947f00b013b7/addons/base/views.py#L53), [here](https://github.com/mfraezz/osf.io/blob/22879db9739afa7abc81a8086d60947f00b013b7/addons/base/views.py#L64), [here](https://github.com/mfraezz/osf.io/blob/22879db9739afa7abc81a8086d60947f00b013b7/addons/base/views.py#L75), [here](https://github.com/mfraezz/osf.io/blob/22879db9739afa7abc81a8086d60947f00b013b7/addons/base/views.py#L83), or [here](https://github.com/mfraezz/osf.io/blob/22879db9739afa7abc81a8086d60947f00b013b7/addons/base/views.py#L95).

Reply:
Ah, thanks